TUTORIAL.rst

```diff
@@ -0,0 +1,187 @@
+PDFmorph Tutorial
+#################
+
+Welcome! This will be a quick tutorial to accquaint users with PDFmorph
+and some of what it can do.
+
+As we described in the README and installation instructions, please make
+sure that you are familiar with working with your command line terminal
+before using this application.
+
+Before you've started this tutorial, please ensure that you've installed
+all necessary software and dependencies.
+
+Basic PDFmorph Workflow
+-----------------------
+
+	1. Open your Terminal or Command Prompt.
+
+	2. It it's not active already, activate your PDFmorph-equipped
+	   conda environment by typing in ::
+
+		source activate <pdfmorph_env>
+
+	   on Linux or ``activate <pdfmorph_env>`` on Windows.
+
+		* If you need to list your available conda environments,
+		  run the command ``conda info --envs`` or
+		  ``conda env list``
+
+        * Run the ``pdfmorph --help`` command and read over the
+          info on that page for a brief overview of some of what we will
+          explore in this tutorial.
+
+	3. Using the ``mkdir`` command, create a directory where you'll
+        store the tutorial PDF files and use the ``cd`` command to change
+        into that directory. You can download the tutorial files
+        :download:`here <tests/testdata/tutorialData.zip`. Then, ``cd``
+        into the ``tutorialData`` directory.
+
+        * The files in this dataset were collected by Soham Banerjee
+          at Brookhaven National Laboratory in Upton, New York.
+
+        * The files are PDF data collected on Iridium Telluride with
+          20% Rhodium Doping (IrRhTe2) with the first file (01) collected
+          at 10K and the last (44) at 300K. The samples increase in
+          temperature as their numbers increase. The "C" in their names
+          indicates that they have undergone cooling.
+
+        * Note that these files have the ``.gr`` extension, which
+          indicates that they are measured PDFs. The ``.cgr`` file
+          extension indicates that a file is a calculated PDF, sich as
+          those generated by the `PDFgui <https://www.diffpy.org/products/pdfgui.html>`_
+          program.
+
+    4. First, we will run the PDFmorph application without any morphing
+       and only using one PDF. Type the following command into your
+       Command Line ::
+
+            pdfmorph darkSub_rh20_C_01.gr darkSub_rh20_C_01.gr
+
+        This should produce two PDF curves which are congruent, resulting
+        in a flat green line underneath them.
+
+    5. Now, we will see PDFmorph run with two different PDFs and no
+       morphing. Type the following command into your command line ::
+
+            pdfmorph darkSub_rh20_C_01.gr darkSub_rh20_C_44.gr
+
+       Without morphing, the difference Rw = 0.407. This indicates that
+       the two PDFs vary drastically.
+
+            * While running the ``pdfmorph`` command, it is important
```

Comment:
please double-check this is right.

The best way to get this across is probably to reiterate here that the command is 
`` pdfmorph <morphed PDF file> <target PDF file> ``